File: win-prog/ProgWinCS/Text and Fonts/BoldAndItalicBigger/BoldAndItalicBigger.cs
//--------------------------------------------------
// BoldAndItalicBigger.cs © 2001 by Charles Petzold
//--------------------------------------------------
using System;
using System.Drawing;
using System.Windows.Forms;

class BoldAndItalicBigger: BoldAndItalic
{
     public new static void Main()
     {
          Application.Run(new BoldAndItalicBigger());
     }
     public BoldAndItalicBigger()
     {
          Text += " Bigger";
          Font = new Font("Times New Roman", 24);
     }
}

File: win-prog/ProgWinVB/Text and Fonts/BoldAndItalicBigger/BoldAndItalicBigger.vb
'----------------------------------------------------
' BoldAndItalicBigger.vb (c) 2002 by Charles Petzold
'----------------------------------------------------
Imports System
Imports System.Drawing
Imports System.Windows.Forms

Class BoldAndItalicBigger
    Inherits BoldAndItalic

    Shared Shadows Sub Main()
        Application.Run(new BoldAndItalicBigger())
    End Sub

    Sub New()
        Text &= " Bigger"
        Font = New Font("Times New Roman", 24)
    End Sub
End Class
